lib/response-handlers/foods-response-handler.js

```diff
@@ -0,0 +1,19 @@
+const url = `https://protected-basin-11627.herokuapp.com/`
+
+const handleResponse = require('../response-handlers/initial-response-handler')
```

Comment:
It looks like you have an initial-response-handler file w/in the response handlers folder.  I understand where you're trying to go with this, but I want to see how it plays out.  I worry that the code might become too dry

Reply:
I was thinking it'd be useful for both of us to have this available within our respective pages. I don't mind extracting this out and putting it back into this response handler file. But for now, I'm also leaning towards seeing how things go when the app becomes more built out.

If at any time we feel like we're getting close to a level organization either of us are uncomfortable with, let's definitely let each other know and talk about it 😄 👍